Assets/Scripts/Game/Automap.cs

```diff
@@ -1500,9 +1503,10 @@ private void DestroyUserMarkerNotes()
         {
             if (gameObjectUserNoteMarkers != null)
             {
+                var meshFilters = gameObjectUserNoteMarkers.GetComponentsInChildren<MeshFilter>();
```

Comment:
Why do you get `meshFilters` here? It's not used in this scope.
Did you mean to `Destroy` those too?

Same for line 1700

Reply:
Those were leftovers from some debugging I was doing. They're removed now.